frame/include/bli_type_defs.h

```diff
@@ -68,8 +70,8 @@ typedef          uint32_t guint_t;
 typedef           int64_t  gint_t;
 typedef          uint64_t guint_t;
 #else
-typedef   signed long int  gint_t;
-typedef unsigned long int guint_t;
+typedef         ptrdiff_t  gint_t;
+typedef            size_t guint_t;
```

Comment:
I opted to use `size_t` and `ptrdiff_t` because they accurately describe the fact that these pointers will be used to address within a single object. Unfortunately, there are some architectures weird enough that `size_t` is not always the same as `uintptr_t` (http://stackoverflow.com/a/14068191/1935144). There's also the fact that MSVC 2008 doesn't have a `stdint.h` by default, so, if/when we have a build system capable of using msvc 2008 for the reference kernels, we'd be stuck providing platform specific definitions for these integers to provide sensible default behavior. It seemed simpler to rely on `stddef.h` instead.